JavaAssignments/src/saket/Ternaryoperator.java

```diff
@@ -0,0 +1,40 @@
+package saket;
+
+public class Ternaryoperator {
+
+	public static void main(String[] args) {
+		
+		
+		
+		int a=1;
+		int b=89;
+		int max;
+		
+		max=(a>b)?a:b;
+		
+		if(a==max)
```

Comment:
You dont need this if condition..

You have already written max = (a>b)?a:b mean if a is bigger then b then max will be a else max is b.

then why need to again write if condition.